Perl/RegExp (Perl angle brackets).sublime-syntax

```diff
@@ -0,0 +1,19 @@
+%YAML 1.2
```

Comment:
I had been confused about how to do interpolation without `push`/`with_prototype`, but I forgot that the `extends` pattern is available now. Exciting!